classes/components/forms/context/PKPEmailSetupForm.inc.php

```diff
@@ -39,6 +41,22 @@ public function __construct($action, $locales, $context) {
 
 		$canEnvelopeSender = \Config::getVar('email', 'allow_envelope_sender');
 
+		$this->addField(new FieldRadioInput('copySubmissionAckPrimaryContact', [
+			'label' => __('manager.setup.notifications.copySubmissionAckPrimaryContact'),
+			'isMultilingual' => false,
+			'options' => [
+				['value' => true, 'label' => __('manager.setup.notifications.copySubmissionAckPrimaryContact.enabled')],
+				['value' => false, 'label' => __('manager.setup.notifications.copySubmissionAckPrimaryContact.disabled')],
+			],
+			'value' => $context->getData('copySubmissionAckPrimaryContact'),
+		]));
```

Comment:
Let's use the following `label`:

> Notify Primary Contact

And the following `description`:

> Send a copy of the submission acknowledgement email to this journal's primary contact.

(Note that this description will need to be specific for OJS/OMP/OPS. Use "journal" for OJS, "press" for OMP, and "preprint server" for OPS.)

Then change the options to the following, using the primary contact email address:

> - Yes, send a copy to nate@example.com.
> - No